crates/katana/node/src/config/mod.rs

```diff
@@ -45,6 +45,9 @@ pub struct Config {
 
     /// Development options.
     pub dev: DevConfig,
+
+    /// Gas L1 sampling options.
+    pub gas_price_worker: Option<GasPriceWorkerConfig>,
```

Comment:
replace this with just the l1 provider url
```suggestion

    pub l1_provider: Option<Url>,
```